src/main/app/claims/converters/address.ts

```diff
@@ -0,0 +1,10 @@
+import { Address as FormAddress } from 'forms/models/address'
+import { Address as AddressDto } from 'claims/models/address'
+
+export function convertAddress (addressForm: FormAddress) {
+
+  const address: AddressDto = new AddressDto()
```

Comment:
Could we not use `*DTO` for consistency please? We don't have this anywhere.

Reply:
ok

Reply:
I meant just the naming convention of course :)